include/llvm/CodeGen/TargetLowering.h

```diff
@@ -3664,6 +3664,9 @@ class TargetLowering : public TargetLoweringBase {
   virtual SDValue LowerToTLSEmulatedModel(const GlobalAddressSDNode *GA,
                                           SelectionDAG &DAG) const;
 
+  virtual SDValue LowerToAndroidEmulatedTLSAddress(SDValue Op, SDValue Result,
+                  SelectionDAG &DAG, bool is64bit) const; // LDC
+
```

Comment:
Wouldn't need to be virtual, right?

Reply:
Not sure, it's a cut-n-paste of `LowerToTLSEmulatedModel` above, which doesn't have multiple class implementations either. I have almost zero knowledge of the LLVM class hierarchy, just followed suit.